packages/build-ui/src/pages/agits/_id/management/collectors/controllers.rs

```diff
@@ -0,0 +1,71 @@
+#![allow(unused)]
+use std::io::Read;
+
+use bdk::prelude::{dioxus_popup::PopupService, *};
+
+use super::models::{Collector, Asset};
+// use super::models::*;
+
+
+#[derive(Debug, Clone, Copy, DioxusController)]
+pub struct Controller{
+    lang: Language,
+    collector: Signal<Vec<Collector>>,
+    asset: Signal<Vec<Asset>>,
+    popup: PopupService,
+
+ 
+}
+impl Controller{
+  pub fn new(lang: Language, agit_id: ReadOnlySignal<i64>)-> Result<Self, RenderError>{
+    let mut popup :PopupService = use_context();
+    let collector = use_signal(||{
+        (1..15)
+        .map(|id| Collector{
+            id: id.to_string(),
+            total_volume: 2.370,
+            total_volume_usd: 8147.63,
+            collector_id: "10FEO!20".to_string(),
+            owned: 2,
+            token_ids: vec!["1234567890".to_string(), "1234567890".to_string()],
+            wallet_address: "0x1234567890abcdef".to_string(),
+            last_activity: "2023-10-01".to_string(),
+            verified: true,
+        })
+        .collect::<Vec<_>>()
+    });
```

Comment:
Now, all APIs has been implemented already.
Use api model to get data and communicate with @RyanCho-0 
- Refer to https://docs.biyard.co/common/models/api-client.html